index.html

```diff
@@ -47,7 +47,7 @@ <h1>Submission Page</h1>
 							You can submit the code in any language.
 							You can upload the file below.</p>
 						<input class="inp1" type="text" name="email" placeholder="Enter Username" required /><br><br><br>
-						<input class="inp2" type="file" name="upload"><br>
+						<input class="inp1" type="file" name="upload"><br>
```

Comment:
`<input class="inp1" type="file" name="upload"><br>` should be replaced by `<input class="inp3" type="file" name="upload"><br>` as `inp2` needs `margin-left: 600px`